.github/workflows/chromatic-pr.yml

```diff
@@ -1,17 +1,66 @@
-name: Chromatic - Release PR opened
+name: Chromatic - Pull Request
 
-on:
-  pull_request:
-    types:
-      - opened
+on: pull_request
 
 jobs:
-  chromatic:
+  chromatic_changesets:
     # Only run this workflow when the Release PR is opened/created.
     if: startsWith(github.head_ref, 'changeset-release/')
-    name: Chromatic - Build on Release PR (PR opened)
+    name: Chromatic - Build on Release PR (changesets)
     uses: ./.github/workflows/chromatic-build.yml
     with:
       target: 'release'
     secrets:
       projectToken: ${{ secrets.CHROMATIC_APP_CODE }}
+
+  # ------- Regular PRs --------
```

Comment:
**note:** Moved the Chromatic build step on regular PRs to the `pull_request` event instead of `push` to be able to get info available for the pull_request context.